File: sort-the-gift-code/__tests__/sortTheGiftCode.test.js
const { sortGiftCode } = require('../sortTheGiftCode');

describe('Given a string of 26 unique character, return these in a alphabetical order', () => {
    test('Given "abcdef", return "abcdef"', () => {
        const string = "abcdef";
        expect(sortGiftCode(string)).toBe("abcdef");
    });
    test('Given "pqksuvy", return "kpqsuvy"', () => {
        const string = "pqksuvy";
        expect(sortGiftCode(string)).toBe("kpqsuvy");
    });
    test('Given "zyxwvutsrqponmlkjihgfedcba", return "abcdefghijklmnopqrstuvwxyz"', () => {
        const string = "zyxwvutsrqponmlkjihgfedcba";
        expect(sortGiftCode(string)).toBe("abcdefghijklmnopqrstuvwxyz");
    });
    test('Given "qgcahkxfljwuv", return "acfghjklquvwx"', () => {
        const string = "qgcahkxfljwuv";
        expect(sortGiftCode(string)).toBe("acfghjklquvwx");
    });
    test('Given "avbncrftxpkdisl", return "abcdfiklnprstvx"', () => {
        const string = "avbncrftxpkdisl";
        expect(sortGiftCode(string)).toBe("abcdfiklnprstvx");
    });
    test('Given "qgdcezofxprtkvbhmnjyilau", return "abcdefghijklmnopqrtuvxyz"', () => {
        const string = "qgdcezofxprtkvbhmnjyilau";
        expect(sortGiftCode(string)).toBe("abcdefghijklmnopqrtuvxyz");
    });
});

//https://www.codewars.com/kata/52aeb2f3ad0e952f560005d3/train/javascript

File: python-venv/sort_the_gift_code_(sort_string_alphabetically)/__tests__/sort_the_gift_code_test.py
from kata_function.sort_the_gift_code import sort_gift_code

def test_1():
    string = "abcdef"
    assert sort_gift_code(string) == "abcdef"

def test_2():
    string = "pqksuvy"
    assert sort_gift_code(string) == "kpqsuvy"

def test_3():
    string = "zyxwvutsrqponmlkjihgfedcba"
    assert sort_gift_code(string) == "abcdefghijklmnopqrstuvwxyz"

def test_4():
    string = "qgcahkxfljwuv"
    assert sort_gift_code(string) == "acfghjklquvwx"

def test_5():
    string = "avbncrftxpkdisl"
    assert sort_gift_code(string) == "abcdfiklnprstvx"

def test_6():
    string = "qgdcezofxprtkvbhmnjyilau"
    assert sort_gift_code(string) == "abcdefghijklmnopqrtuvxyz"

#https://www.codewars.com/kata/52aeb2f3ad0e952f560005d3/train/python